README.md

```diff
@@ -1,11 +1,11 @@
 # Data Source JS
 
-Data Source is a JS library meant to help developers access Movable Ink Data Sources and the raw responses associated with their API integrations.
+Data Source is a JS library meant to help developers access Movable Ink Data Sources and the raw responses associated with that Data Source.
 
 ## Installation
 Add data sources to your package.json file. In `dependencies` include the following:
 ```
-"data-source.js": "https://github.com/movableink/data-source.js.git#v0.1.0"
+"data-source.js": "https://github.com/movableink/data-source.js.git#v0.1.6"
```

Comment:
this should be `v0.1.0`?

Reply:
Ah good catch. I didn't see that we fixed it!

Reply:
Also updated the package.json.